README.md

```diff
@@ -156,12 +158,23 @@ The following [dispatchers](./telemetry/producer.go#L10-L19) are supported
 * Google pubsub: Along with the required pubsub config (See ./test/integration/config.json for example), be sure to set the environment variable `GOOGLE_APPLICATION_CREDENTIALS`
 * ZMQ: Configure with the config.json file.  See implementation here: [config/config.go](./config/config.go)
 * Logger: This is a simple STDOUT logger that serializes the protos to json.
-  
+
 >NOTE: To add a new dispatcher, please provide integration tests and updated documentation. To serialize dispatcher data as json instead of protobufs, add a config `transmit_decoded_records` and set value to `true` as shown [here](config/test_configs_test.go#L186)
 
 ## Reliable Acks
 Fleet telemetry can send ack messages back to the vehicle. This is useful for applications that need to ensure the data was received and processed. To enable this feature, set `reliable_ack_sources` to one of configured dispatchers (`kafka`,`kinesis`,`pubsub`,`zmq`) in the config file. Reliable acks can only be set to one dispatcher per recordType. See [here](./test/integration/config.json#L8) for sample config.
 
+## Detecting Vehicle Connectivity Changes
+On the vehicle, Fleet telemetry client behave similarly to how the connectivity engine for vehicle commands. Therefore we can use Fleet telemetry connectivity event to assume when a vehicle is online. Note that it is a proxy, but if configured properly fleet telemetry connectivity time should match vehicle connectivity state in 99%+. To enable connectivity events simply add the `connectivity` records in the list of events in config.json file:
+
+  ```
+    "records": {
```

Comment:
nit: maybe refer to integration test config file?
https://github.com/teslamotors/fleet-telemetry/blob/main/test/integration/config.json#L49